### PR TITLE
feat: update text for the inquiry flow

### PR DESCRIPTION
**Pull request by egdbear**
The type of this PR is: **Feat**

This PR solves [DIA-402]

### Description
This PR updates the text copy as specified in the ticket.

Before: 
Screenshot 2024-06-06 at 11 40 56 AM

After: 

Screenshot 2024-06-06 at 11 41 04 AM

Screenshot 2024-06-06 at 12 05 31 PM

Screenshot 2024-06-06 at 12 19 16 PM

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DIA-402]: https://artsyproduct.atlassian.net/browse/DIA-402?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by araujobarret — DISMISSED** [review]

File: src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx [hunk 1]
```diff
@@ -13,6 +22,17 @@ const SettingsEditProfileRoute: React.FC<SettingsEditProfileRouteProps> = ({
 }) => {
   return (
     <GridColumns>
+      <Column span={8}>
+        <Box p={2} mb={1} bg="black5">
+          <Flex>
+            <InfoIcon />
+            <Text variant="sm-display" marginLeft={"5px"}>
```
Comment (dzucconi) [inline src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:29]: Use themed values.

File: src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx [hunk 2]
```diff
@@ -13,6 +22,17 @@ const SettingsEditProfileRoute: React.FC<SettingsEditProfileRouteProps> = ({
 }) => {
   return (
     <GridColumns>
+      <Column span={8}>
```
Comment (dzucconi) [inline src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: Both the box and the flex aren't needed. Set display flex on the column, remove the mb as well.

Comment (dzucconi) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: Notice also instance how that icon shrunk.

Comment (dzucconi) [inline src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]:
```tsx
<Column span={8} bg="black5" p={2} display="flex" gap={1}>
        <InfoIcon flexShrink={0} />

        <Text variant="sm-display">
          The information you provide here will be shared when you contact a
          gallery or make an offer.
        </Text>
      </Column>
```

However! Wondering if this should actually be a [banner](https://palette-storybook.artsy.net/?path=/story/components-banner--default)?

Comment (egdbear) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: My understanding is that Banner components are typically used for status updates or confirmations (confirmed in the code). I'm not sure if we should use it here. Is this a design question, or can we proceed with the existing approach? Open to suggestions!

Comment (dzucconi) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: Yeah, that was also my question. I think typically we use the [Message](https://palette-storybook.artsy.net/?path=/story/components-message--default) for this sort of thing, though I do see the need for an icon slot.

Comment (dzucconi) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: Anyway, fine with existing approach (with the CSS edits I propose) for the time being.

Comment (egdbear) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx:25]: 👁️

Comment (egdbear) [reply src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx]: sounds good!

**Review by dzucconi — DISMISSED** [review]

**Pull request merged**